public/src/components/amounts/amountInput.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import {createStyles, Theme, withStyles, WithStyles} from "@material-ui/core";
+
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+
+const styles = ({ palette, spacing, mixins }: Theme) => createStyles({
+  amountInput: {
+    display: 'inline',
+    marginLeft: 'auto',
+    paddingLeft: '8px',
+    borderLeft: `1px dotted ${palette.grey['300']}`
+  },
+  amountTextField: {
+    width: '40px'
```

Comment:
having played around with it, this is a bit narrow and cuts off the numbers... maybe 80px?